codebuild-terraform-apply/data_buildassets.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_iam_role" "execution_role" {
+  name = "CodePipelineDJExecutionRole"
```

Comment:
This should be the execution role name and in the new execution role I've given it a more sensible name

Reply:
https://github.com/alphagov/cyber-security-terraform/blob/f3e233ff0fdfb59b4f8ca8bddbc4398da357ff01/service/codepipeline-build-assets/terraform/modules/buildassets/iam.tf#L2